documentation/modules/migrating-virtual-machines-cli.adoc

```diff
@@ -51,13 +51,14 @@ type: Opaque
 stringData:
   user: <user> <1>
   password: <password> <2>
-  cacert: <{rhv-short}_ca_certificate> <3>
+  cacert: | <3>
+    <{rhv-short}_ca_certificate>
   thumbprint: <vcenter_fingerprint> <4>
 EOF
 ----
-<1> Specify the base64-encoded vCenter user or the {rhv-short} {manager} user.
-<2> Specify the base64-encoded password.
-<3> {rhv-short} only: Specify the base64-encoded CA certificate of the {manager}. You can retrieve it at `https://<www.example.com>/ovirt-engine/services/pki-resource?resource=ca-certificate&format=X509-PEM-CA`.
+<1> Specify the vCenter user or the {rhv-short} {manager} user.
+<2> Specify the user password.
+<3> {rhv-short} only: Specify the CA certificate of the {manager}. You can retrieve it at `https://<fqdn>/ovirt-engine/services/pki-resource?resource=ca-certificate&format=X509-PEM-CA`.
```

Comment:
@mguetta1 Do you approve?

Reply:
@istein1 @mguetta1 
1. Should line 60 read "Specify the user password" or "Specify the admin password"?
2. Does the answer to question apply to both MTV 2.2 and MTV 2.3?

Reply:
@RichardHoch I think it fits for both 2.2 and 2.3
The thing is that regarding **VMware only** - in 2.2 the requirement is to use admin user (mentioned in (1) in 2.2 doc)
and for 2.3 it can be any user (with required permissions for MTV migration - mentioned in #282 PR)
In my opinion, "Specify the user password" is OK for both versions, as long we mention in (1) in 2.2 doc: "Specify the vCenter **admin** user or..."